### PR TITLE
fix: update health routes to be codable

### PR DESCRIPTION
**Pull request by KyeMaloy97**
Updated the health routes file to leverage codable routing, no regression as the behavior is the same as before, it returns two items if theres an issue and one if there isn't.

### CODE REVIEW

File: refresh/templates/common/RouteTests.swift [hunk 1]
```diff
@@ -58,6 +58,32 @@ class RouteTests: XCTestCase {
...
+    func testHealthRoute() {
+        
+        let printExpectation = expectation(description: "The /health route will print UP, followed by a timestamp.")
+        
+        URLRequest(forTestWithMethod: "GET", route: "health")?
+            .sendForTestingWithKitura { data, statusCode in
+                if let getResult = String(data: data, encoding: String.Encoding.utf8) {
+                    XCTAssertEqual(statusCode, 200)
+                    XCTAssertTrue(getResult.contains("UP"))
+                    XCTAssertTrue(getResult.contains("2017") || getResult.contains("2018") || getResult.contains("2019"))
```
Comment (christiancompton) [inline refresh/templates/common/RouteTests.swift]: Can we use a more dynamic timestamp check here?  This will be a pain to maintain.

Comment (KyeMaloy97) [reply refresh/templates/common/RouteTests.swift]: Updated to dynamically get the current year and make a check that its the same as returned in the test.

File: refresh/templates/common/RouteTests.swift [hunk 2]
```diff
@@ -58,8 +58,37 @@ class RouteTests: XCTestCase {
...
+            .sendForTestingWithKitura { data, statusCode in
+                if let getResult = String(data: data, encoding: String.Encoding.utf8) {
+                    XCTAssertEqual(statusCode, 200)
+                    XCTAssertTrue(getResult.contains("UP"))
+                    let date = Date()
+                    let calendar = Calendar.current
+                    let yearString = String(describing: calendar.component(.year, from: date))
+                    XCTAssertTrue(getResult.contains(yearString)
+                } else {
+                    XCTFail("Return value from /health was nil, or was not UP, or had no timestamp.")
```
Comment (EnriqueL8) [inline refresh/templates/common/RouteTests.swift]: I believe this error message doesn't give correct information about the error. 
It is better to put the errors inside the asserts such as : 
`XCTAssertEqual(statusCode, 200, "Error")`
And set this XCTFail to "The conversion to String(..) failed" something along those lines which is what the `if ` is checking for.

File: refresh/templates/common/RouteTests.swift [hunk 3]
```diff
@@ -58,8 +58,37 @@ class RouteTests: XCTestCase {
...
+        
+        URLRequest(forTestWithMethod: "GET", route: "health")?
+            .sendForTestingWithKitura { data, statusCode in
+                if let getResult = String(data: data, encoding: String.Encoding.utf8) {
+                    XCTAssertEqual(statusCode, 200)
+                    XCTAssertTrue(getResult.contains("UP"))
+                    let date = Date()
+                    let calendar = Calendar.current
+                    let yearString = String(describing: calendar.component(.year, from: date))
+                    XCTAssertTrue(getResult.contains(yearString)
```
Comment (EnriqueL8) [inline refresh/templates/common/RouteTests.swift]: I believe there is a parentheses missing here

**Comment by christiancompton** [comment]
@KyeMaloy97 Can you address the concerns above?

**Review by EnriqueL8 — APPROVED** [review]

**Pull request merged**